problems/10-publish-again/index.js

```diff
@@ -18,7 +18,7 @@ exports.verify = function (args, cb) {
 
   var __ = this.__
   var pkg = require(path.join(shop.cwd(), 'package.json'))
-  var data = require(path.join(shop.datadir, 'registry', pkg.name, 'body.json'))
+  var data = require(path.join(shop.datadir, 'registry/store', pkg.name, 'package.json'))
```

Comment:
I think we should keep same style for similar lines. I prefer:
https://github.com/workshopper/how-to-npm/blob/964688c5b525cea41304cb5cc4aff251b96d5330/problems/11-dist-tag/index.js#L22

Reply:
Agree, make sense, but I guess you meant `data` rather  `body` . Unless you want I change it also to `body`.